CMS/translations.py

```diff
@@ -681,6 +681,10 @@
         'en': 'Next page',
         'cy': 'Y dudalen nesaf'
     },
+    'no_courses_found': {
+        'en': 'No courses found, remove or change some filters',
+        'cy': 'Ni chafwyd hyd i gyrsiau dileu newid rhai hidlwyr.'
```

Comment:
If the Welsh translation is to match the English, it should have a comma after the word `gyrsiau`. However, if I was being pedantic, I'd query whether the English/Welsh is correct in the first place - shouldn't a hyphen be used instead?

Personally, I'd use 
```
No courses found - remove or change some filters
Ni chafwyd hyd i unrhyw cyrsiau - dileu neu newid rhai hidlwyr
```